content/en/docs/instrumentation/js/serverless.md

```diff
@@ -173,3 +173,123 @@ function.
 
 You should now be able to view traces produced by OpenTelemetry from your Lambda
 function in the backend!
+
+## GCP function
+
+The following shows how to instrument [http triggered function](https://cloud.google.com/functions/docs/writing/write-http-functions) using the Google Cloud Platform (GCP) UI.
+
+### Creating function
+
+Login to GCP and select the project where your function should be placed. From serverless select Cloud Functions. Select create function and select environment. This example uses second generation. 
+
+### Setup environment variable for otelwrapper 
+
+Add the environment variable `NODE_OPTIONS` with the following value:
+
+```shell
+--require ./otelwrapper.js
+```
+
+### Select runtime
+Select Node.js version 16.
+
+### Establish otel wrapper
+
+Create a new file called `otelwrapper.js`, that will be used to instrument your service. Please make sure that you provide a `SERVICE_NAME` and that you set the `<address for your backend>`
```

Comment:
> provide a `SERVICE_NAME`

Where in code would this go? I see this:

```
[SemanticResourceAttributes.SERVICE_NAME]: "<your function name>",
```

But that's proposing that we set the service name to be the name of the function.

Reply:
```suggestion
Create a new file called `otelwrapper.js`, that will be used to instrument your service. Please make sure that you provide your function name as `SERVICE_NAME` and that you set the `<address for your backend>` in the appropriate lines.
```